python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -331,3 +332,71 @@ def get_units(value):
         return _unit_map[value.lower()]
 
     return value
+
+
+class Container(object):
+    def __init__(self, arg):
+        self.arg = arg
+
+
+class DateOffset(pd.DateOffset, Container):
+    def __init__(self, n=1, normalize=False, **kwds):
```

Comment:
We need a docstring somewhere in here and we should note what is not yet supported

Reply:
Added this